app/components/collections/contributor_component.rb

```diff
@@ -0,0 +1,13 @@
+# frozen_string_literal: true
+
+module Collections
+  # Component for editing collection contributors
+  class ContributorComponent < ApplicationComponent
```

Comment:
These don't seem necessary to componentize. And if you do, they can just be be subclasses of TextFieldComponent instead of rendering a TextFieldComponent.